.github/ISSUE_TEMPLATE/team-update.md

```diff
@@ -1,21 +1,47 @@
 ---
-name: Team updates
-about: Giving a team update
-title: "Tech Team Update: [DATE]"
-labels: team-process
+name: "🙌 Team sync"
+about: Syncronize the team's goals and actions
+labels: team-sync
+title: "Team Sync - {{ date | date('dddd, MMMM Do') }}"
 ---
 
-Hey @2i2c-org/tech-team - time to fill in some updates about what you've been up to the last couple of weeks!
-Can folks fill out the [HackMD](https://hackmd.io/i2Siurp1TkmPYgn3ZgxFQw) with their own updates? ✨ 
+This is a 2i2c Tech Team sync 🎉🎉🎉! This is a way for 2i2c Team Members to triage work, point out issues that require attention, and make sure we are working well as a team.
```

Comment:
Can we 'ping' the tech team here somehow automatically as well so it shows up in github notifications?